In [2]:
!pip install opencv-python

Defaulting to user installation because normal site-packages is not writeable


In [1]:
import cv2

In [2]:
import numpy as np

In [18]:
input = cv2.imread('images/car1.jpeg') # to read an image  and specifying the path of the image 


In [13]:
cv2.imshow('Hey There!',input) # displays the image in the specified window with name
# first para : image window
# second para: image variable

#waitKey allows us to input info when a image window is open 
#By leaving it blank it just waits for any key to be pressed before 
#continuing by replacing numbers(except 0 ), we can specify a delay for 
#how long you keep the window open (time in ms)
cv2.waitKey(0)

#this closes all open windows , Failure to places this will cause your program to hang 
cv2.destroyAllWindows()


In [19]:
print(input.shape)

(170, 297, 3)


height : 170 , width : 297 , channel : 3(RGB)

### Shape gives the dimension of the image array
The 2D dimensions are 830 pixels in high bv 1245 pixels wide . The '3L' means that there are 3 other components (RGB) that makes up this image 

In [20]:
#dimension of the image 

print('Height of the image:',int(input.shape[0]),'pixels')
print('Width of the image:',int(input.shape[1]),'pixels')

Height of the image: 170 pixels
Width of the image: 297 pixels


### To Save the images edited in open cv 

In [21]:
cv2.imwrite('output.jpg',input)


True